spec/controllers/download_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe DownloadController, type: :controller do
```

Comment:
Can we avoid stubbing out RSF and running the importer as part of the setup, and just create some database fixtures using FactoryBot? Or if not, maybe extract this setup to a common test helper (I think there are a few others that follow the same pattern)?

In this test we're not actually testing the contents of the spreadsheet, so a lot of the code in this file is incidental what we're testing.

On that note, maybe its worth creating a unit test that parses the resulting ODS and checks the data and headings are correct? I'm not too worried about this though.

Reply:
@MatMoore Yes, I agree ideally we would parse the contents of the ODS file unfortunately this would require another test dependency that can parse this format so I'm not sure it's worth the effort.

I will look at moving the test boilerplate out somewhere.

Reply:
OK, I moved out the RSF importing now in: https://github.com/openregister/registers-frontend/pull/496/commits/d1c02054b6e41300d6b81f12396254f0f291b597